docs/install.md

```diff
@@ -1,77 +1,220 @@
+
+### 4. Create a GCE Instance as Worker
+*   Create a new Managed Instance Group from the newly created Instance Template
+*   In your worker VM, add a new custom metadata key `startup-script-url`
+    pointing to `gs://$your_bucket/scripts/start-wrapper.sh`
+*   TODO(beamcodeup): Update this section when Issue #132 is resolved.
+*   Upon start, your VM should mount your GCS Bucket, and copy the start scripts
+    into the home directory of the Turbinia user and will then start the
+    Turbinia worker
+*   ***NOTE:*** If the GCS FUSE is used for log files, update the Turbinia config
+    file with a new `LOG_FILE` path to avoid writing to the server logs.
+
+## Core Installation Steps
+
+### 1. Preparation Work
+
+*   Install dependencies
+    *   `sudo apt-get install python-dev build-essential python-setuptools
+        python-pip python-virtualenv liblzma-dev git`
+*   Create a turbinia user with password-less sudo access **(IMPORTANT)**
+    *   Add your turbinia user to the `disk` group so that you will have
+        permissions to read attached Persistent Disks
+    *   `sudo adduser turbinia disk`
+    *   Add the turbinia user to `/etc/sudoers` by inserting this line:
+        `turbinia ALL=(ALL:ALL) NOPASSWD: ALL`
+*   Log in as turbinia
+    *   `su - turbinia`
+*   Continue to [Google Cloud SDK](#2-google-cloud-sdk)
+
+### 2. Google Cloud SDK
+
+*   Install
+    [google-cloud-sdk](https://cloud.google.com/sdk/docs/quickstart-linux)
+*   Create a [scoped service
+    account](https://cloud.google.com/compute/docs/access/service-accounts)
+    (this is the best option) with the following roles:
+    *   `Cloud Datastore User`: Used by PSQ to store result data, and in the
+        future by the Task Manager to store queriable task data
+    *   `Pub/Sub Editor`: Used by clients to talk to Turbinia, and by the Task
+        Manager to talk to workers
+    *   `Storage Object Admin` and `Storage Legacy Bucket Reader`: Only required
+        on the GCS bucket used by Turbinia, if any. See [GCP Turbinia](#gcp-turbinia)
+        for details
+    *   `Compute Instance Admin`: Used to list instances and to attach disks to
+        instances
+    *   `Service Account User`: Used when attaching disks
+    *   `Cloud Functions Developer`: Used by turbiniactl to query task status
+*   Create a new key for your service account, and then point to it with an
+    environment variable:
+    *   `export
+        GOOGLE_APPLICATION_CREDENTIALS="/home/turbinia/turbinia-service-account-creds.json"`
+*   Add the service account to the gcloud auth **(RECOMMENDED)**
+    *   `gcloud auth list`
+    *   `gcloud auth activate-service-account
+        --key-file=$GOOGLE_APPLICATION_CREDENTIALS`
+*   Alternately you can run Turbinia under your own credentials **(NOT
+    RECOMMENDED)**
+    *   Run `gcloud auth login` (may require you to copy/paste url to browser)
+    *   Or run `gcloud auth application-default login`
+*   Create (once) and activate Virtualenv
```

Comment:
Maybe just move the virtualenv related instructions to the "Inside the virtualenv" section?

Reply:
done